filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -75,11 +75,15 @@ static void clearGlError() noexcept {
 
 // ---------------------------------------------------------------------------------------------
 
-PlatformEGL::PlatformEGL() noexcept = default;
+PlatformEGL::PlatformEGL(EGLDisplay display) noexcept {
+    mEGLDisplay = display;
```

Comment:
nit: this could go into initializer list

Reply:
1b30b28